event-log/src/main/scala/io/renku/eventlog/events/categories/statuschange/ProjectEventsToNewUpdater.scala

```diff
@@ -131,23 +133,26 @@ private class ProjectEventsToNewUpdaterImpl[F[_]: Async: Logger](
       .void
   }
 
-  private def removeProjectDeletingEvents(project: Project) = measureExecutionTime {
-    SqlStatement(name = "project_to_new - awaiting_deletions removal")
+  private def removeProjectEvents(project: Project, status: EventStatus) = measureExecutionTime {
+    SqlStatement(name = Refined.unsafeApply(s"project_to_new - ${status.value.toLowerCase()} removal"))
       .command[EventStatus ~ projects.Id](
         sql"""DELETE FROM event
               WHERE status = $eventStatusEncoder AND project_id = $projectIdEncoder
         """.command
       )
-      .arguments(EventStatus.Deleting ~ project.id)
+      .arguments(status ~ project.id)
       .build
       .mapResult {
         case Completion.Delete(count) => count
         case _                        => 0
       }
   }
 
-  private def cleanUpDeletingMetrics(project: Project, removeProjectDeletingEvents: Int) =
-    Kleisli.liftF(deletingEventGauge.update((project.path, -removeProjectDeletingEvents.toDouble)))
+  private def cleanUpAwaitingDeletionMetrics(project: Project, eventsCount: Int) =
+    deletingEventGauge.update((project.path, -eventsCount.toDouble))
```

Comment:
I think there is a gauge for the awaiting deletion event as well. Not sure if we should combine these two.